### ISSUE TITLE
Collab with RichEditorView

### ISSUE THREAD

**Issue by YoomamaFTW** [nnhubbard/ZSSRichTextEditor#258]
Hi RichTextEditor community! I'm from [YoomamaFTW/RichEditorView](https://github.com/YoomamaFTW/RichEditorView), which is a revised and iOS 13 compatible RichEditorView.

I'm looking for those willing to contribute some JS help. Currently, I am aiming to make our repositories better than Apple's by supporting all functionalities and adding more. Currently, all our editors don't have a couple features such as drawing, document scan, checkboxes, and tables.

Actually, I just implemented checkboxes in my repo and want to share it with you guys. But for the others, we should collaborate on the JS since that is the most difficult (and by far my unhappiest) coding times. Lemme know if you're interested.

**Comment by fooshyangsong** [comment]
Hello @YoomamaFTW , I had used  cjwirth/RichEditorView before. Does your side added on feature like toolbarItemSelected tint color like ZSSRichTextEditor ?

**Comment by YoomamaFTW** [comment]
No. I didn't realize people used the default toolbar, but I can add a feature like that.
